src/components/EventListLoading/styles.js

```diff
@@ -4,12 +4,7 @@ const styles = () => ({
     justifyContent: 'center',
     alignItems: 'center',
     width: '100%',
-    margin: 20,
-  },
-  loading: {
-    width: '5rem',
-    height: '5rem',
-    marginTop: 360,
+    margin: '20px',
```

Comment:
use 20 (no 'px' unless needed), its preferred in JSS

Reply:
I tried without px, but it won't work as I expected